script/Arcueid/Day 13/13_00_ARC13_2.txt

```diff
@@ -126,66 +134,71 @@ Perhaps everything until now has been a pointless struggle with no meaning or sa
 [sha:f25e6f516d8e88d29e39c95960f3a7e29f3f4e9a]{
 -- Page 223, Offset 32275.
 -- 彼女にとって、それはもう過去の問題だった。
-To her, that is already a problem in the past.
+All of that is already in the past for her.
 }
 [sha:359c205417ce739a2e8d6c1a3744076f1b22fa2b]{
 -- Page 223, Offset 32276.
 -- 自分の事も、<仇|ロ><敵|ア>の事も、取るに足りない確執だ。
-Her own circumstances and Roa are nothing more than a worthless feud.
+The feud between her and Roa doesn't even merit a thought.
```

Comment:
I'm not much for the expression "merit a thought". Maybe something a little different:
> Arcueid, Roa, and their trivial rivalry.

Reply:
Seems weird for her to think of herself in third person? What about
> The feud between herself and Roa seems trivial.

I also prefer `feud` to `rivalry` because they aren't really competing for something, and it's a bit more aggressive.

Reply:
swap the clauses and I'm cool with that. I went with Arcueid originally because for whatever reason, the jibun here felt more omniscient.
> The trivial rivalry between herself and Roa.